prime-router/src/main/kotlin/TranslatorConfiguration.kt

```diff
@@ -0,0 +1,104 @@
+package gov.cdc.prime.router
+
+import com.fasterxml.jackson.annotation.JsonCreator
```

Comment:
I'm puzzled as to why we had to have a new concept of "Configuration" (or Template, as you call it in your proposal).   Seems like we could just allow all schemas to be parameterized with values found in the organizations data, then just have a redox_covid_standard.schema and hl7_covid_standard.schema.

The references to `covid` embedded in the code below are a sign that this is still kinda custom code.